bindings/CMakeLists.txt

```diff
@@ -39,14 +39,14 @@ endif()
 
 
 # list of all modules that will be included in libdnf bindings
-list(APPEND SWIG_LIBDNF_MODULES base)
+list(APPEND SWIG_LIBDNF_MODULES base common conf logger rpm)
 
 
 # list of all modules that will be included in libdnf-cli bindings
 list(APPEND SWIG_LIBDNF_CLI_MODULES progressbar)
 
 
-add_subdirectory(go)
-add_subdirectory(perl5)
+#add_subdirectory(go)
+#add_subdirectory(perl5)
```

Comment:
Why are you disabling this? 😢

Reply:
There is not `std_set.i` for Perl and Go language. It is not a problem of SWIG but Perl and Go languages do not have native support for `set`. So I disable they for a while and started creating bindings for Python. It is on good way. I have prepared several unit tests of Python API. Then I worked on Ruby bindings. It is on good way too. I rewrote the Python unit tests to the Ruby. And they are working.
And now I'm thinking about enabling Perl and Go and solving the problem with `set` type.

Reply:
Awesome!